spdx-utils/src/test/kotlin/SpdxDeclaredLicenseMappingTest.kt

```diff
@@ -66,11 +66,9 @@ class SpdxDeclaredLicenseMappingTest : WordSpec({
         }
 
         "not contain plain SPDX license ids" {
-            assertSoftly {
-                SpdxDeclaredLicenseMapping.mapping.forEach { (declaredLicense, _) ->
-                    "\"$declaredLicense\" maps to ${SpdxLicense.forId(declaredLicense)}" shouldBe
-                            "\"$declaredLicense\" maps to null"
-                }
+            SpdxDeclaredLicenseMapping.mapping.keys.forAll { declaredLicense ->
+                "\"$declaredLicense\" maps to ${SpdxLicense.forId(declaredLicense)}" shouldBe
+                        "\"$declaredLicense\" maps to null"
```

Comment:
I hope I'm not mistaken that `declaredLicense shouldBe null` gives good enough output. Given that I think this `assert message magic` should be removed.

Reply:
Done.